bridge/client/sdp.js

```diff
@@ -379,6 +387,11 @@ if (typeof(SDP) == "undefined")
         var sdpText = fillTemplate(templates.sdp, sdpObj);
         sdpText = fillTemplate(sdpText, sdpObj.originator);
 
+        if (!sdpObj.noTrickle)
+            var iceTrickleLine = "a=ice-options:trickle\r\n";
+
+        sdpText = fillTemplate(sdpText, { "trickleLine": iceTrickleLine });
```

Comment:
Maybe it's because I am more used to C-like variable scoping but I find the above `if () var` style to be a bit difficult to read and understand. I guess `iceTrickleLine` will be undefined in the case that `sdpObj.noTrickle === true`, right? Is fillTemplate with an undefined trickleLine doing what you want it to? Is there some way to rewrite this to make it more clear/explicit.